src/test/kotlin/com/terraformation/backend/accelerator/db/VoteStoreTest.kt

```diff
@@ -221,141 +223,42 @@ class VoteStoreTest : DatabaseTest(), RunsAsUser {
-      insertVote(
-          project2,
-          phase0,
-          user200,
-          votes[VoteKey(user200, project2, phase0)],
-          createdTime = clock.instant)
-      insertVote(
-          project2,
-          phase1,
-          user100,
-          votes[VoteKey(user100, project2, phase1)],
-          createdTime = clock.instant)
-      insertVote(
-          project2,
-          phase1,
-          user200,
-          votes[VoteKey(user200, project2, phase1)],
-          createdTime = clock.instant)
+    fun `throws exception on delete if no permission to update votes `() {
+      val projectId = insertProject(participantId = inserted.participantId)
+      val phase: CohortPhase = CohortPhase.Phase1FeasibilityStudy
+      val newUser = insertUser(100)
+      val vote = VoteOption.No
+      insertVote(projectId, phase, newUser, vote)
 
-      store.delete(project2)
+      every { user.canUpdateProjectVotes(projectId) } returns false
 
-      assertEquals(
-          listOf(
-              ProjectVotesRow(
-                  projectId = project1,
-                  phaseId = phase0,
-                  userId = user100,
-                  voteOptionId = votes[VoteKey(user100, project1, phase0)],
-                  conditionalInfo = null,
-                  createdBy = user.userId,
-                  createdTime = clock.instant,
-                  modifiedBy = user.userId,
-                  modifiedTime = clock.instant,
-              ),
-              ProjectVotesRow(
-                  projectId = project1,
-                  phaseId = phase0,
-                  userId = user200,
-                  voteOptionId = votes[VoteKey(user200, project1, phase0)],
-                  conditionalInfo = null,
-                  createdBy = user.userId,
-                  createdTime = clock.instant,
-                  modifiedBy = user.userId,
-                  modifiedTime = clock.instant,
-              ),
-              ProjectVotesRow(
-                  projectId = project1,
-                  phaseId = phase1,
-                  userId = user100,
-                  voteOptionId = votes[VoteKey(user100, project1, phase1)],
-                  conditionalInfo = null,
-                  createdBy = user.userId,
-                  createdTime = clock.instant,
-                  modifiedBy = user.userId,
-                  modifiedTime = clock.instant,
-              ),
-              ProjectVotesRow(
-                  projectId = project1,
-                  phaseId = phase1,
-                  userId = user200,
-                  voteOptionId = votes[VoteKey(user200, project1, phase1)],
-                  conditionalInfo = null,
-                  createdBy = user.userId,
-                  createdTime = clock.instant,
-                  modifiedBy = user.userId,
-                  modifiedTime = clock.instant,
-              )),
-          projectVotesDao.findAll())
+      org.junit.jupiter.api.assertThrows<AccessDeniedException> {
```

Comment:
Add an import for `assertThrows` to avoid having to fully qualify it here and below (IntelliJ should have an option for this in the context menu).